src/componentsLinks.ts

```diff
@@ -21,7 +21,7 @@ export const registerComponentsLinks = () => {
             const text = document.getText()
             const matches: RegExpMatchArray[] = [
                 ...text.matchAll(/(import (.+)\s+from )(['"].+['"])/g),
-                ...text.matchAll(/(\s*(?:let|const)\s*(\w+).*\((?:\/\*.*\*\/\s*)?)(['"].+['"])\)/g),
+                ...text.matchAll(/(\s*(?:let|const)\s*(\w+).+?import\((?:\/\*.*?\*\/\s*)?)(['"].+['"])\)\)/g),
```

Comment:
Why did you add extra `)\`? I see only one literally opening bracket for me it failing on 101 with the example you provided. Did you test it?

Reply:
I'm insane. Didn't manage to paste from 101 correctly. Thanks for being attentive